lib/new_relic/distributed_trace.ex

```diff
@@ -70,20 +70,22 @@ defmodule NewRelic.DistributedTrace do
   def track_transaction(context, transport_type: type) do
     context
     |> assign_transaction_guid()
-    |> maybe_generate_sampling()
+    |> maybe_generate_sampled_priority()
     |> maybe_generate_trace_id()
     |> report_attributes(transport_type: type)
     |> report_attributes(:w3c)
     |> convert_to_outbound()
     |> set_tracing_context()
   end
 
-  def maybe_generate_sampling(%Context{sampled: nil, priority: nil} = context) do
+  def maybe_generate_sampled_priority(%Context{sampled: sampled, priority: priority} = context)
```

Comment:
? this fixes it?

Reply:
The change is that this function clause will get triggered if _either_ are `nil` instead of when _both_ are nil...